web_development_101/the_front_end/project_html_css.md

```diff
@@ -84,6 +84,8 @@ There are a couple of ways to go about doing this, but the simplest is this:
 Submit a pull request with a link to your solution in this [file](https://github.com/TheOdinProject/curriculum/blob/master/web_development_101/the_front_end/project_html_css.md) when your project is complete. See the section on [Contributing](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md) for how.
 
 * Add your solution below this list
+https://iacobssorin.github.io/google-homepage/
```

Comment:
[Iacob Sorin solution (Easy Version)](https://github.com/Iacobssorin/google-homepage) - [View in browser](https://iacobssorin.github.io/google-homepage/)